app/views/dashboards/followers.html.erb

```diff
@@ -44,20 +44,7 @@
       <% else %>
         <div class="p-9 crayons-card crayons-card--secondary align-center fs-l">You don't have any followers yet...</div>
       <% end %>
+      <%= render "sublist" %>
     </div>
   <div>
-
-  <div class="dashboard-container crayons-layout" id="user-dashboard">
-
-    <% if @organizations && (params[:which].blank? || params[:which] == "organization") %>
```

Comment:
Cleanup looks good, but any reason we're removing the organization dropdown?

Reply:
It has been moved elsewhere when I was doing Dashboard updates, just forgot to remove it from here :)